app/main.js

```diff
@@ -4,5 +4,5 @@ import App from './App';
 
 ReactDOM.render(
   <App />,
-  document.body
+  document.body.appendChild(document.createElement('div'))
```

Comment:
Why is this needed?

Reply:
On latests React versions, rendering into the `body` directly throws the following error:

`Warning: render(): Rendering components directly into document.body is discouraged, since its children are often manipulated by third-party scripts and browser extensions. This may lead to subtle reconciliation issues. Try rendering into a container element created for your app.`